### PR TITLE
Formatted elementwise isfinite

### LINKED ISSUE #1939: isfinite

### PR DESCRIPTION
**Pull request by hattiq**
Closes #1939

### CODE REVIEW

**Comment by hattiq** [comment]
reason and fix for failing docstring test #1944

**Comment by RickSanchezStoic** [comment]
Kindly fix the lint errors.

**Comment by RickSanchezStoic** [comment]
You still have the lint errors and `test-docstrings / run-docstring-tests `  also need to go green, kindly check and fix the same. Thanks!

**Comment by hattiq** [comment]
Thanks @RickSanchezStoic for your comments, if you please look into the details of the linting error or the test-docstring error are not because of `isfinite` i.e. subject of the PR. 
I ran `flake8 $(git ls-files '*.py')` locally and it passed.
Linting error is in `ivy/backend_handler`:
```
Run flake8 $(git ls-files '*.py')
ivy/backend_handler.py:1[7](https://github.com/unifyai/ivy/runs/7429901236?check_suite_focus=true#step:5:8)4:89: E501 line too long (90 > 88 characters)
```
Docstring error is in `repeat`:
```
ivy_tests/test_docstrings.py::test_docstrings[torch]
  /ivy/ivy/__init__.py:585: UserWarning: Output is unequal to the docstrings output: repeat
    warnings.warn(warning_message, stacklevel=stacklevel)
```

**Comment by RickSanchezStoic** [comment]
Hi @hattiq kindly fix the `lint errors , thanks!

File: ivy/functional/ivy/elementwise.py [hunk 1]
```diff
@@ -2388,6 +2388,85 @@ def isfinite(
...
+    {
+        a: ivy.array([True, True]),
+        b: ivy.array([True, False])
+    }
+    
+    With :code:`ivy.Container` static method:
+    
+    >>> x = ivy.Container(a=ivy.array([0., 999999999999]),\
+                          b=ivy.array([float('-0'), ivy.nan]))
+    >>> y = ivy.Container.static_isfinite(x)
```
Comment (RickSanchezStoic) [inline ivy/functional/ivy/elementwise.py:2464]: the static method examples should go only in the docstrings of `static_isfinite`

**Review by RickSanchezStoic — CHANGES_REQUESTED** [review]

File: ivy/functional/ivy/elementwise.py (hunk 1 above)
Comment (hattiq) [reply ivy/functional/ivy/elementwise.py]: done.

**Comment by RickSanchezStoic** [comment]
Looks good to me!

**Pull request merged**